app/presenters/two_factor_authentication/set_up_webauthn_platform_selection_presenter.rb

```diff
@@ -0,0 +1,43 @@
+module TwoFactorAuthentication
+  class SetUpWebauthnPlatformSelectionPresenter < SetUpSelectionPresenter
+    def method
+      :webauthn_platform
+    end
+
+    def initialize(user:, configuration: nil)
+      @user = user
+      @configuration = configuration
+    end
```

Comment:
I'd suggest reordering these methods so that the constructor is first.

```suggestion
    def initialize(user:, configuration: nil)
      @user = user
      @configuration = configuration
    end

    def method
      :webauthn_platform
    end
```

Reply:
Got that reordered.